docs/source/_json/batching.json

```diff
@@ -0,0 +1,48 @@
+GET /plone/folder/@search?b_size=5&sort_on=path
+Accept: application/json
+
+HTTP 200 OK
+content-type: application/json
+
+{
+  "@id": "http://localhost:55001/plone/folder/@search?sort_on=path", 
```

Comment:
Right. My feeling is that we should remove the sort_on param from the canonical top-level id as well. Do we have any hypermedia controls for changing the sort order? Have you looked into that?